src/referencePackages/src/system.runtime.interopservices/4.1.0/Customizations.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp1.1' or '$(TargetFramework)' == 'netstandard1.5'">
+    <!-- Needs additional APIs that don't exist in the contract. -->
+    <Compile Include="$(MSBuildThisFileDirectory)Customizations.cs" />
+  </ItemGroup>
```

Comment:
It might be easier to just always import Customizations.cs files if they exist and put these TFM checks directly into that file. The roslyn repo often prefers that over conditioning in the project file and that worked great for them.

This would mean that the Customizations.cs would start with a `#if NETCOREAPP1_1 || NETSTANDARD1_5` line.

Reply:
Hmm, I originally thought this pattern would provide more flexibility but I don't think that is necessarily the case.  The pattern you suggest reduces the onboarding maintenance/ease of use.  I will make this change.

Reply:
Addressed in https://github.com/dotnet/source-build-reference-packages/pull/1140/commits/394218c4ceb60e337d4076883846f791088f9444